src/components/scenes/newAccount/NewAccountTosScene.tsx

```diff
@@ -215,16 +217,24 @@ export const NewAccountTosScene = (props: NewAccountTosProps) => {
       })
       await setTouchOtp(account, dispatch)
 
-      dispatch({
-        type: 'NAVIGATE',
-        data: {
-          name: 'newAccountReview',
-          params: {
-            ...route.params,
-            account
+      if (lightAccount) {
+        if (onLogin == null)
+          throw new Error(
+            'NewAccountTosScene: onLogin required for light accounts'
+          )
+        onLogin(account)
```

Comment:
`if (onLogin != null) onLogin(account)`, and ditch the `throw`

Reply:
When is it valid to have that behavior?